### DISCUSSION TITLE
Long-running restic commands via backrest possible?

### DISCUSSION THREAD

**Discussion by oliver-la** [garethgeorge/backrest#482] [Q&A] [answered]
The web ui currently allows to run a custom restic command, with the caveat that it's only intended for commands that complete in a short time.

I'd like to run a prune with a couple extra flags. Since it will take a long time, I probably(?) can't do it via the Web UI.

Is there an easier way than to call the restic binary directly and re-supply all environment variables and path to repo config? If not, what should the command look like?

**Comment by garethgeorge** [comment, marked as answer]
Hey, this is definitely something on my mental roadmap -- but if this is something you're interested in feel free to create a feature request ticket, it helps me with prioritization and w/perspective on how users are thinking about a feature.

My current idea is to eventually introduce a new operation type + task type for custom commands that will be tracked in the operation log (and support all the normal task features like cancellation, output and log tracking, etc). I'll likely implement some small condensed display for recently run custom command output in the Run Command view.

Edit: you're right to think that (at least right now) run command is pretty unsuited to long running operations.
